src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -1513,6 +1514,11 @@ export const CURVE_MAINNET_INFOS: { [name: string]: CurveInfo } = {
         pool: CURVE_POOLS.bLUSD,
         gasSchedule: 390e3,
     }),
+    [CURVE_POOLS.crvfrax]: createCurveExchangePool({
```

Comment:
can you add a link to simbot in the description?

Reply:
done